# <font color='black'>Project Title : ***Netflix Price Predction*** </font>

# <font color='blue'> Description</font>


We will use historical stock price data for Netflix to train machine learning models to predict future stock prices. The dataset includes columns such as Date, Open, High, Low, Close, Adjusted Close (Adj Close), and Volume. The 'Close' price will be our target variable, as it represents the final trading price for each day and is a key metric for investors aiming to maximize profit by buying low and selling high. The other columns ('Open', 'High', 'Low', 'Adj Close', 'Volume') serve as input features since they influence the 'Close' price and can improve the model's predictive accuracy.

Why 'Close' Price?
The 'Close' price is crucial because it reflects the stock's final value at the end of the trading day, providing insight into the stock’s daily performance. This price is affected by daily fluctuations captured in the 'Open', 'High', 'Low', and 'Adj Close' prices, as well as the trading volume. Therefore, predicting the 'Close' price helps investors decide the optimal times to buy or sell.

By using these features, our model aims to accurately predict the future 'Close' price of Netflix, allowing for informed investment decisions.

![Netflix image](image2.jpg)




## <font color ='Green'>1) Loading Libraries</font>

In [16]:
import numpy as np
import pandas as pd

In [22]:
df=pd.read_csv("netflix.csv")

In [24]:
df.head()

Date      Open      High       Low     Close  Adj Close     Volume
0  2002-05-23  1.156429  1.242857  1.145714  1.196429   1.196429  104790000
1  2002-05-24  1.214286  1.225000  1.197143  1.210000   1.210000   11104800
2  2002-05-28  1.213571  1.232143  1.157143  1.157143   1.157143    6609400
3  2002-05-29  1.164286  1.164286  1.085714  1.103571   1.103571    6757800
4  2002-05-30  1.107857  1.107857  1.071429  1.071429   1.071429   10154200

In [27]:
df.shape

(5044, 7)

**Observation:**
<font color='Red'>1) We have Total 5044 number of Rows and 7 Columns in the Dataset.</font>